citi-onboarding/Eventos-CITi
# Feat/add-insync-to-back-gitignore

**What I did:**

Add .insync to the .gitignore file at the server folder.

**How to test:**
1. Open VSCode in "Eventos-CITi"
2. Open "server" and ".gitignore"
3. Check if the .insync line is at the .gitignore file
